st2common/st2common/models/api/base.py

```diff
@@ -283,6 +282,7 @@ def cast_func(value):
             try:
                 result = f(*args, **kwargs)
             except TypeError as e:
+                LOG.debug(traceback.format_exc())
```

Comment:
Is this needed here or you just add it to debug things locally?

Just wondering since this exception should get logger elsewhere - in one layer above where this function is used. If it's not, then I'm fine with this change.